docs/man/rpm-man-template.scd

```diff
@@ -72,15 +79,15 @@ See *rpm.common*(8) for the options common to all *rpm* executables.
 *--do-this*
 	Do Y instead of default action.
 
-*-o* _ARG_,
-*--output*=_ARG_
+*-o* _FILE_,
+*--output*=_FILE_
 	Use file _ARG_ for output.
```

Comment:
Seems like this was forgotten to be updated.

Reply:
(One of those cases where just pushing a fixup! is easier than comment & explain, but meh) :smile:

Reply:
ARGh :facepalm: Yep, fixed :laughing: